spec/dummy/Gemfile.lock

```diff
@@ -4,7 +4,7 @@ PATH
     react_on_rails (1.0.0.pre)
       connection_pool
       execjs (~> 2.5)
-      rails (>= 4.0)
+      rails (>= 3.2)
```

Comment:
@justin808 I'm pretty sure it was a glitch somehow, since I didn't want to change it and master didn't change it. As you can see in that link, my computer was constantly changing it back to that. I restarted and it stopped doing it so that I could set it straight and force push a new commit that doesn't include that change (so the log will be cleaner than having a "Undo change Gemfile.lock" commit).